PSReadLine/Options.cs

```diff
@@ -167,6 +168,14 @@ private void SetOptionsInternal(SetPSReadLineOption options)
                     }
                 }
             }
+            if (options._terminateStragglers.HasValue)
+            {
+                Options.TerminateStragglers = options.TerminateStragglers;
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    PlatformWindows.SetTerminateStragglers(Options.TerminateStragglers);
+                }
```

Comment:
>} [](http://example.com/codeflow?start=16&length=1)

If not on Windows, do we want to notify the user that this doesn't actually do anything?

Reply:
(actually, the Options-setting line should be inside the `if` block, too, and that might be enough)